src/tests/simulation/src/cluster.rs

```diff
@@ -87,6 +87,25 @@ pub struct Configuration {
 
     /// Path to etcd data file.
     pub etcd_data_path: Option<PathBuf>,
+
+    /// Queries to run per session.
+    pub per_session_queries: Arc<Vec<String>>,
```

Comment:
cc @yezizp2012 and @shanicky. I have included this field which allows us to set `STREAMING_ENABLE_ARRANGEMENT_BACKFILL` to false whenever we start a new session.

This is used for when we set `STREAMING_ENABLE_ARRANGEMENT_BACKFILL` to true by default, e.g. during testing. Otherwise scaling tests can break due to assumptions for `NoShuffle`.

Reply:
> This is used for when we set `STREAMING_ENABLE_ARRANGEMENT_BACKFILL` to true by default, e.g. during testing.

Could you show me where it's set to true by default?

Reply:
This commit reverts it: https://github.com/risingwavelabs/risingwave/pull/14148/commits/4884da3949be14f30129d9ee6b2a084608b47fbc.

We just:
1. Manually modify the code to set it to `true` by default, then all MVs and Sinks will use it instead of backfill.
2. In our test suite we can comprehensively test `ArrangementBackfill` now, without having to call `SET ARRANGEMENT BACKFILL=true`.
3. For scaling case, there are several tests that make assumptions around `NoShuffle`. We just `SET ARRANGEMENT_BACKFILL=false` before these tests, so it will use normal `backfill`, since (1) has been done.
4. That way with or w/o 1, the tests will still pass.
5. They will also be backwards compatible, if we choose to permanently set `ArrangementBackfill` to true by default in the future.